diff --git a/node_modules/lodash.debounce/index.js b/node_modules/lodash.debounce/index.js
index ac5707d..7447acd 100644
--- a/node_modules/lodash.debounce/index.js
+++ b/node_modules/lodash.debounce/index.js
@@ -38,7 +38,7 @@ var freeGlobal = typeof global == 'object' && global && global.Object === Object
 var freeSelf = typeof self == 'object' && self && self.Object === Object && self;
 
 /** Used as a reference to the global object. */
-var root = freeGlobal || freeSelf || Function('return this')();
+var root = freeGlobal || freeSelf;
 
 /** Used for built-in method references. */
 var objectProto = Object.prototype;
