diff --git a/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js b/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js
index 8091399..48e8321 100644
--- a/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js
+++ b/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js
@@ -325,7 +325,11 @@ function record(options) {
             }, hooks);
         };
         iframeManager.addLoadListener(function (iframeEl) {
-            handlers_1.push(observe_1(iframeEl.contentDocument));
+            try {
+                handlers_1.push(observe_1(iframeEl.contentDocument));
+            } catch (error) {
+                console.warn('error in rrweb iframe observer', error);
+            }
         });
         var init_1 = function () {
             takeFullSnapshot();
