diff --git a/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js b/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js
index 8091399..48e8321 100644
--- a/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js
+++ b/node_modules/rrweb/es/rrweb/packages/rrweb/src/record/index.js
@@ -325,7 +325,11 @@ function record(options) {
             }, hooks);
         };
         iframeManager.addLoadListener(function (iframeEl) {
-            handlers_1.push(observe_1(iframeEl.contentDocument));
+            try {
+                handlers_1.push(observe_1(iframeEl.contentDocument));
+            } catch (error) {
+                console.warn('error in rrweb iframe observer', error);
+            }
         });
         var init_1 = function () {
             takeFullSnapshot();
diff --git a/node_modules/rrweb/typings/index.d.ts b/node_modules/rrweb/typings/index.d.ts
index 6b6d18d..31456a0 100644
--- a/node_modules/rrweb/typings/index.d.ts
+++ b/node_modules/rrweb/typings/index.d.ts
@@ -2,7 +2,7 @@ import record from './record';
 import { Replayer } from './replay';
 import { _mirror } from './utils';
 import * as utils from './utils';
-export { EventType, IncrementalSource, MouseInteractions, ReplayerEvents, } from './types';
+export { EventType, IncrementalSource, MouseInteractions, ReplayerEvents, eventWithTime, recordOptions } from './types';
 declare const addCustomEvent: <T>(tag: string, payload: T) => void;
 declare const freezePage: () => void;
 export { record, addCustomEvent, freezePage, Replayer, _mirror as mirror, utils, };
